src/parallant.erl

```diff
@@ -51,16 +49,16 @@ test(Algorithm) ->
            Steps :: pos_integer(),
            Log :: boolean()) -> ok.
 test(Algorithm, Seed, Width, Height, Depth, NAgents, Steps, Log) ->
-    %%     io:format("ListBasedAgentsImpl:~n"),
-    %%     random:seed(Seed),
-    %%     start(Width, Height, Depth, NAgents, Steps, [{algorithm, Algorithm},
-    %%                                                  {agents, agents},
-    %%                                                  {log, Log}]),
-    %%     io:format("Gb_treeBasedAgentsImpl:~n"),
-    %%     random:seed(Seed),
-    %%     start(Width, Height, Depth, NAgents, Steps, [{algorithm, Algorithm},
-    %%                                                  {agents, agents_gbt},
-    %%                                                  {log, Log}]),
+    %%         io:format("ListBasedAgentsImpl:~n"),
+    %%         random:seed(Seed),
+    %%         start(Width, Height, Depth, NAgents, Steps, [{algorithm, Algorithm},
```

Comment:
Line 54 is too long:     %%         start(Width, Height, Depth, NAgents, Steps, [{algorithm, Algorithm},.